code-block/src/components/CodeEditor/CodeEditor.tsx

```diff
@@ -169,3 +141,57 @@ export const CodeEditor: FunctionComponent<{
     </div>
   )
 }
+
+const FieldSet: FunctionComponent<
+  React.DetailedHTMLProps<
+    React.FieldsetHTMLAttributes<HTMLFieldSetElement>,
+    HTMLFieldSetElement
+  >
+> = (props) => (
+  <fieldset
+    css={css({
+      display: 'flex',
```

Comment:
do you think there is a way to extract this css...I see there are some imported values so thats probably why you did not put into css file

Reply:
Extract where? I prefer to keep the styles within the module where they're being used.

Reply:
I extracted the header into a new component now; CodeEditorHeader.tsx

Reply:
This project uses emotion, because it need to dynamically update styles with query selectors (using `:nth-of-type`, so the `style` property won't work)